disposable_email_blacklist.conf

```diff
@@ -1942,6 +2059,7 @@ spamfree24.info
 spamfree24.net
 spamfree24.org
 spamgoes.in
+spamgourmet.com
```

Comment:
`spamgourmet.com` is on the whitelist as it requires a username and password to access.